From c9888a6a2bc368199a952e40347769266ff8eb40 Mon Sep 17 00:00:00 2001
From: Fei Jiang <fei.jiang@intel.com>
Date: Thu, 26 Jul 2018 13:43:28 +0800
Subject: [PATCH 1410/1667] drm/i915/gvt: enable plane wm pvmmio level through
 enable_pvmmio param

plane wm update pvmmio level is 0x4, need set it in enable_pvmmio for both
SOS and UOS kernel driver.
Patch for both SOS and UOS.

V2: use PVMMIO_PLANE_WM_UPDATE bit definition to improve readability

Signed-off-by: Fei Jiang <fei.jiang@intel.com>
Reviewed-by: Min He <min.he@intel.com>
Reviewed-by: Zhao Yakui <yakui.zhao@intel.com>
---
 drivers/gpu/drm/i915/i915_params.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_params.c b/drivers/gpu/drm/i915/i915_params.c
index 1cfbdf1..f94ff99 100644
--- a/drivers/gpu/drm/i915/i915_params.c
+++ b/drivers/gpu/drm/i915/i915_params.c
@@ -73,7 +73,8 @@ struct i915_params i915_modparams __read_mostly = {
 	.inject_load_failure = 0,
 	.enable_dpcd_backlight = false,
 	.enable_gvt = false,
-	.enable_pvmmio = PVMMIO_ELSP_SUBMIT | PVMMIO_PLANE_UPDATE,
+	.enable_pvmmio = PVMMIO_ELSP_SUBMIT | PVMMIO_PLANE_UPDATE |
+			 PVMMIO_PLANE_WM_UPDATE,
 	.enable_gvt_oos = 1,
 	.enable_conformance_check = true,
 	.disable_gvt_fw_loading = true,
-- 
2.7.4

